ports/atmel-samd/boards/pyportal_titano/mpconfigboard.h

```diff
@@ -0,0 +1,40 @@
+#define MICROPY_HW_BOARD_NAME "Adafruit PyPortal Titano"
+#define MICROPY_HW_MCU_NAME "samd51j20"
+
+#define CIRCUITPY_MCU_FAMILY samd51
+
+// This is for Rev B
```

Comment:
I think this is copy pasta from a previous board, and should probably be removed. The same comment is in:
```
pyportal/mpconfigboard.h
itsybitsy_m4_express/mpconfigboard.h
pybadge/mpconfigboard.hpybadge_airlift/mpconfigboard.h
```

Reply:
Thanks for spotting that, resolved in 5a884d